### ISSUE TITLE
How to exclude unneeded built-in commands?

### ISSUE THREAD

**Issue by Cepr0** [fonimus/ssh-shell-spring-boot#56]
How to exclude unneeded built-in commands?
For example jvm-env, jvm-properties etc?
I tried to do the following but without luck:
```java
@ComponentScan(excludeFilters = @ComponentScan.Filter(type= FilterType.ASSIGNABLE_TYPE, value = {
        SystemCommand.class,
        StacktraceCommand.class,
        ThreadCommand.class
}))
@SpringBootApplication
public class Application {
    //...
}
```

**Comment by fonimus** [comment]
I think the best way would be for me to add properties in order to not create bean at all.
I will added in the next release, thank you for posting issue !

**Comment by fonimus** [comment]
It is now included in master (exclude command by **ssh.shell.default-commands.<command>=false**).
It will be available in next release (1.1.6).
